src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Startup.cs

```diff
@@ -116,12 +116,10 @@ public void ConfigureServices(IServiceCollection services)
 
 #endif
 #if (OrganizationalAuth)
-            services.AddControllersWithViews(options =>
+
+            services.AddAuthorization(options =>
             {
-                var policy = new AuthorizationPolicyBuilder()
-                    .RequireAuthenticatedUser()
-                    .Build();
-                options.Filters.Add(new AuthorizeFilter(policy));
+                options.FallbackPolicy = options.DefaultPolicy;
```

Comment:
Could we include the snippet that explains what this does:

> // By default, all incoming requests will be authorized according to the default policy

It's not common to include comments in code, but fallback policy is a little new and has no documentation. So putting it here might help a bit

Reply:
I added the comment.

I'm also planning on adding some docs about this to as recommended in the original issue.